src/examples/files.jl

```diff
@@ -52,3 +52,12 @@ dirresponse(f) =
             div(".box", table([tr(td(".file", filelink(f, x)),
                                   td(".size", string(filesize(joinpath(f, x)))))
                                for x in ["..", readdir(f)...]]))))
+
+const ASSETS_DIR = "assets"
+function packagefiles(dirs=true)
+    absdir(req) = Pkg.dir(req[:params][:pkg], ASSETS_DIR)
+    branch(req -> Mux.validpath(absdir(req), joinpath(req[:path]...), dirs=dirs),
+           req -> Mux.fresp(joinpath(absdir(req), req[:path]...)))
```

Comment:
Removed the `Mux.`s

There's a check [here](https://github.com/JuliaWeb/Mux.jl/blob/6b00aed6d70fc71f7488c3b5a4e596ccd65aab7c/src/examples/files.jl#L11) that is intended to contain access to the root directory.